docs/advanced-usage/index.md

```diff
@@ -26,3 +26,7 @@
 **Containers**
 
 - [Docker](containers/docker) useful tips for Docker containerised environments
+
+**User Quotas**
+
+- [User Quotas](user-quotas) to deal with limits on the resources available
```

Comment:
- "User quotas" (BrE rather than AmE spelling of title word case)
- to inspect CPU and Disk resource quota limits and manage their use